tensorboard/plugins/histogram/tf_histogram_dashboard/histogramCore.ts

```diff
@@ -121,14 +137,15 @@ export function intermediateToD3(
       }
       bucketIndex++;
     }
-    return {x: binLeft, dx: binWidth, y: binY};
-  });
+    d3HistogramBins.push({x: binLeft, dx: binWidth, y: binY});
+  }
+  return d3HistogramBins;
 }
 
 export function backendToVz(histograms: BackendHistogram[]): VzHistogram[] {
   const intermediateHistograms = histograms.map(backendToIntermediate);
-  const minmin = d3.min(intermediateHistograms, (h) => h.min);
```

Comment:
Just noticed, but this change could be reverted now (they can stay `const`).

Reply:
Changed back, thanks!